geodatasets/lib.py

```diff
@@ -9,6 +9,8 @@
 import json
 import uuid
 
+from shapely import GeometryType
```

Comment:
We can't do this. This would add shapely as a dependency, which I'd like to avoid here.

We can simply just list allowed options here.

Reply:
Also, we do not distinguish between Polygon and MultiPolygon, so this list is not necessarily valid.

Reply:
sure